t/perf/opcount.t

```diff
@@ -781,4 +781,11 @@ test_opcount(0, "builtin::floor is replaced with direct opcode",
                     floor    => 1,
                 });
 
+test_opcount(0, "builtin::tainted is replaced with direct opcode",
+                sub { builtin::floor($0); },
```

Comment:
Looks like changing this got missed when copying the test.

Reply:
Good spot, fixed.